src/Clock.tsx

```diff
@@ -0,0 +1,69 @@
+/* eslint-disable no-console */
+import React from 'react';
+
+const formatAMPM = (date: Date): string => {
+  let hours = date.getHours();
+  const ampm = hours >= 12 ? 'AM' : 'PM';
+
+  hours %= 12;
+  const min = date.getMinutes().toString().padStart(2, '0');
+  const sec = date.getSeconds().toString().padStart(2, '0');
+  const strTime = `${hours}:${min}:${sec} ${ampm}`;
+
+  return strTime;
+};
```

Comment:
```suggestion
  componentDidMount() {
    this.timerId = window.setInterval(() => {
      this.setState({ date: new Date() });
      console.info(this.state.date.toUTCString().slice(-12, -4));
    }, 1000);
  }
```

This will pass your tests, you didn't check the README, bad Samir!

Reply:
vauvv. this is really weird since I don't have these lines on my local.. it looks like I didn't add and commit the latest code from my local :)